pkg/server/server.go

```diff
@@ -59,6 +59,9 @@ type Config struct {
 
 	UpstreamBundle bool
 
+	// Skip agent id validation in node attestation
```

Comment:
I think we can yank this comment? The same one appears below in the actual configurable that does this